### DISCUSSION TITLE
FP: HOw-to keeping selected node in the display area when unfolding branches

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1863] [Help] [answered]
Hi All,  I do not find a solution for keeping the selected node in the display area when unfolding big branches, somebody may suggest how to set it?

**Comment by abc16361** [comment]
I would suggest using a filtering script to bring them together if other nodes are not necessary to show up.

**Comment by euu2021** [comment]
my selected node stays fixed in view when I unfold branches:
![javaw_5CXAGSlAy2](https://github.com/freeplane/freeplane/assets/77707706/d8411c0a-4400-4a2c-becf-a588454233a5)

Does your FP behave differently?

**Reply by Illegit** [reply]
Hi euu202, this is exactly what I need, may you share your settings? Thanks.

**Reply by euu2021** [reply, marked as answer]
I don't know which is the relevant setting, but it's probably one of these:

![image](https://github.com/freeplane/freeplane/assets/77707706/a9cc70f7-f6bc-4ad3-b0e6-9d306dedd0cd)

![image](https://github.com/freeplane/freeplane/assets/77707706/b40e7479-f69d-44fa-b16d-2b74e0a15454)

**Reply by Illegit** [reply]
Thank you, it is working: this flag"Freeplane keeps selected node visible" is tricking this job. I utilize your settings with success.
